### PR TITLE
[Ubuntu 18.04] Removing duplicate repos - dotnetcore-skd.sh

### PR DESCRIPTION
**Pull request by andy-mishechkin**
According to https://github.com/actions/virtual-environments-internal/issues/131, dotnetcore-sdk.sh has been unified for using in both Ubuntu 16/18 deployments.
Also code for additional repository options has been moved from dotnetcore-sdk.sh to repos.sh

### CODE REVIEW

File: .vscode/tasks.json [hunk 1]
```diff
@@ -39,4 +39,4 @@
             "problemMatcher": "$pester"
         }
 	]
```
Comment (maxim-lobanov) [inline .vscode/tasks.json]: Could you please revert this file? It looks like it contains only noise changes

File: images/linux/scripts/base/repos.sh [hunk 2]
```diff
@@ -9,7 +9,11 @@ LSB_RELEASE=$(lsb_release -rs)
 # Install Microsoft repository
 wget https://packages.microsoft.com/config/ubuntu/$LSB_RELEASE/packages-microsoft-prod.deb
 dpkg -i packages-microsoft-prod.deb
-apt-get update
+apt-get install apt-transport-https
```
Comment (sergey-akhalkov) [inline images/linux/scripts/base/repos.sh]: Should we add `-y` flag here?

File: images/linux/scripts/base/repos.sh [hunk 3]
```diff
@@ -9,7 +9,11 @@ LSB_RELEASE=$(lsb_release -rs)
...
 wget https://packages.microsoft.com/config/ubuntu/$LSB_RELEASE/packages-microsoft-prod.deb
 dpkg -i packages-microsoft-prod.deb
-apt-get update
+apt-get install -y apt-transport-https
 
 # Install Microsoft GPG public key
 curl -L https://packages.microsoft.com/keys/microsoft.asc | apt-key add -
+curl https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > microsoft.gpg
+mv microsoft.gpg /etc/apt/trusted.gpg.d/microsoft.gpg
+sh -c 'echo "deb [arch=amd64] https://packages.microsoft.com/repos/microsoft-ubuntu-bionic-prod bionic main" > /etc/apt/sources.list.d/dotnetdev.list'
```
Comment (al-cheb) [inline images/linux/scripts/base/repos.sh]: Why do you want again broken ubuntu 16.04 to add repo from 18.04?

File: images/linux/scripts/base/repos.sh [hunk 4]
```diff
@@ -9,7 +9,11 @@ LSB_RELEASE=$(lsb_release -rs)
 # Install Microsoft repository
 wget https://packages.microsoft.com/config/ubuntu/$LSB_RELEASE/packages-microsoft-prod.deb
 dpkg -i packages-microsoft-prod.deb
```
Comment (al-cheb) [inline images/linux/scripts/base/repos.sh]: Repo `https://packages.microsoft.com/repos/microsoft-ubuntu-bionic-prod bionic main` - already present in packages-microsoft-prod.deb

File: .github/workflows/stale.yml [hunk 5]
```diff
@@ -13,7 +13,7 @@ jobs:
     - uses: actions/stale@v1
       with:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
-        stale-issue-message: 'This issue has not had any activity in 30 days, if you still require assistance please remove the `no-issue-activity` label'
+        stale-issue-message: 'This issue has not had any activity for 45 days and will be closed in 45 days if there continues to be no activity.'
         stale-issue-label: 'no-issue-activity'
-        days-before-stale: 30
-        days-before-close: 14
+        days-before-stale: 45
+        days-before-close: 45
```
Comment (sergey-akhalkov) [inline .github/workflows/stale.yml]: I believe your PR should no contain these changes, could you please update your branch with the latest changes from master?

**Pull request closed without merging**